### DISCUSSION TITLE
Importancia de Git y GitHub en Redes

### DISCUSSION THREAD

**Discussion by Milyy31** [hackatomik/SISCOx256#22] [Preguntas Profesionales | Carrera]
¿Es, o podría llegar a ser indispensable Git y GitHub en la carrera de Redes?

**Comment by quantumquirkxyz** [comment]
Hola, el Git y GitHub pueden ser súper útiles en la carrera de Redes, que mezcla tanto el manejo de hardware como de software. Estas herramientas te ayudan a llevar un control de los cambios en tus configuraciones, scripts y aplicaciones de red, y facilitan la colaboración en proyectos. Git te permite guardar versiones de tu trabajo y volver atrás si algo sale mal, mientras que GitHub es genial para trabajar en equipo, revisar código y mantener todo bien documentado. En el mundo de la automatización y la integración, estas herramientas son clave para gestionar y desplegar configuraciones de manera ordenada. Aunque no siempre son indispensables, definitivamente pueden hacer que tu trabajo en redes sea mucho más eficiente y organizado.

**Reply by tapilew** [reply]
Probablemente en DevOps o SRE?
